smallslive/templates/account/ajax_login.html

```diff
@@ -41,8 +41,7 @@
 </div>
 <script>
     loginModal = document.getElementById('login-modal')
```

Comment:
`loginModal` is undeclared and I don't think it's necessary.
This should be enough.
```javascript
$('#login-modal .close-button').click(() => { $('#logIn').modal('toggle') })
```

Reply:
Done